src/modules/Schedule/components/ColorPicker/utils/getNewColor.ts

```diff
@@ -0,0 +1,25 @@
+import { Course } from '@thinc-org/chula-courses'
+import { difference, random } from 'lodash'
+
+import { CourseCartItem } from '@/store'
+
+import { OTHER_COLORS } from '../constants'
+
+export function getNewColor(currentItems: CourseCartItem[], newCourse: Course) {
```

Comment:
Can we JSdoc the reason we need this function (for when user add course first time and for when user sync with previously uploaded gDrive )?

Reply:
Done
Should be more clear now